foundation/minutes/2022-12-05.html

```diff
@@ -0,0 +1,199 @@
+---
+layout: foundation_minutes
+date: "2022-12-05"
+type: "foundation-minutes"
+order: 207
+sitemap: false
+CSS: foundation-minutes.css
+---
+
+<p style="text-align: center;"><span id="docs-internal-guid-165a598f-7fff-fb98-8397-0ad425ba14c2"><span style="border: none; display: inline-block; overflow: hidden; width: 150px; height: 90px;"><img style="margin-left: 0px; margin-top: 0px;" src="https://lh5.googleusercontent.com/A1p6tjgDDSRxs8jpUMxmV8zkEeVQy5zCDolvMc7-tYndL3nnirnWAHNMf7selVpSuNcAhCn0m9kBVY35TR_zGnT0dEs8P5U2VqxReuI9cecMKV5H0jhXctdmnhVR6WRH0fvwQaGvUdSu3Upfzp8y7jI" width="150" height="90" /></span></span></p>
```

Comment:
I don't think we should be linking to external images like this that might go away. There's already a [Foundation logo in the website](https://github.com/OpenDDS/OpenDDS/blob/gh-pages/foundation/images/opendds-foundation-stacked.png). It's different layout, but if we want to use this one specifically then we should add it as well. 

Also for consistency, If we wanted to make it part of all minutes it should be added to the template for the page instead of the individual minutes pages.